pkg/api/serialization_test.go

```diff
@@ -405,9 +406,15 @@ func TestSpecificKind(t *testing.T) {
 	}
 }
 
+// Keep this in sync with the respective upstream set
+var nonInternalRoundTrippableTypes = sets.NewString("List", "ListOptions", "PodExecOptions", "PodAttachOptions")
```

Comment:
Sorry, this isn't obvious to me.  These are somehow broken?  Also, anything that needs to stay in sync should have a unit test to make sure it stays in sync.